DepositeCalcTests/Tests/LoginPageTests.cs

```diff
@@ -0,0 +1,134 @@
+
+            // Act
+            passworldFld.SendKeys("newyork1");
+            loginBtn.Click();
+            Thread.Sleep(500);
+
+            // Assert
+            Assert.AreEqual("Incorrect user name!", errMessage.Text);
+            driver.Quit();
+        }
+
+        [Test]
+        public void LoginWithEmptyFieldsTest()
+        {
+            // Arrange
+            ChromeOptions options = new ChromeOptions { AcceptInsecureCertificates = true };
+            IWebDriver driver = new ChromeDriver(options);
+
+            driver.Url = "https://localhost:5001/";
+            IWebElement loginBtn = driver.FindElement(By.XPath("//button[@id='login']"));
+            IWebElement errMessage = driver.FindElement(By.Id("errorMessage"));
+
+            // Act
+            loginBtn.Click();
+            Thread.Sleep(500);
+
+            // Assert
+            Assert.AreEqual("User not found!", errMessage.Text);
+            driver.Quit();
+        }
+
+        [Test]
+        public void LoginFieldSignatureTest()
+        {
+            // Arrange
+            ChromeOptions options = new ChromeOptions { AcceptInsecureCertificates = true };
+            IWebDriver driver = new ChromeDriver(options);
+
+            driver.Url = "https://localhost:5001/";
+            IWebElement loginFldSignature = driver.FindElement(By.XPath("//th[@class='user']"));
+
+            // Assert
+            Assert.AreEqual("User", loginFldSignature.Text);
+            driver.Quit();
+        }
+
+        [Test]
+        public void PasswordFieldSignatureTest()
+        {
+            // Arrange
+            ChromeOptions options = new ChromeOptions { AcceptInsecureCertificates = true };
+            IWebDriver driver = new ChromeDriver(options);
+
+            driver.Url = "https://localhost:5001/";
+            IWebElement passwordFldSignature = driver.FindElement(By.XPath("//th[@class='pass']"));
+
+            // Assert
+            Assert.AreEqual("Password", passwordFldSignature.Text);
+            driver.Quit();
+        }
+
+        [Test]
+        public void ValidLoginTest()
+        {
+            // Arrange
+            ChromeOptions options = new ChromeOptions { AcceptInsecureCertificates = true };
+            IWebDriver driver = new ChromeDriver(options);
+
+            driver.Url = "https://localhost:5001/";
+            IWebElement loginFld = driver.FindElement(By.Id("login"));
+            IWebElement passworldFld = driver.FindElement(By.Id("password"));
+            IWebElement loginBtn = driver.FindElement(By.XPath("//button[@id='login']"));
+            String currentURL = driver.Url;
+
+            // Act
+            loginFld.SendKeys("test");
+            passworldFld.SendKeys("newyork1");
+            loginBtn.Click();
+
+            // Assert
+            Assert.IsTrue(driver.FindElement(By.Id("amount")).Displayed);
+            Assert.AreNotEqual(currentURL, driver.Url);
```

Comment:
I think it is better to verify the current URL is equal to '.../Calculator'

Reply:
Got it!
In my opinion, we should also update the README.MD file:
row 10 -  change * Login filed, labeled 'User' to  * Login filed, labeled 'User:'
row 11 -  change * Password filed, labeled 'Password'change to  * Password filed, labeled 'Password:'